completions/completions_v2.py

```diff
@@ -331,28 +337,31 @@ def _run_complete():
 
     def on_text_command(self, view, command_name, args):
 
-        logger.debug("text command, command: {}, args: {}".format(command_name, args))
-        is_new_line_inserted = command_name  == "insert" and args["characters"] == '\n'
- 
+        logger.debug("text command, command: {}, args: {}".format(
+            command_name, args))
+        is_new_line_inserted = command_name == "insert" and args["characters"] == '\n'
+
         if command_name == "replace_completion_with_next_completion":
             self._replace_completion_with_next_completion = True
 
-        if command_name in ["left_delete"] :
+        if command_name in ["left_delete"]:
             self._left_deleted_selection = view.substr(view.sel()[0])
-            self._left_deleted_character = view.substr(max(view.sel()[0].end() - 1, 0))
-            
-        if command_name in [ "left_delete", "commit_completion", "insert_best_completion", "replace_completion_with_next_completion", "toggle_comment", "insert_snippet"] or is_new_line_inserted :
+            self._left_deleted_character = view.substr(
+                max(view.sel()[0].end() - 1, 0))
+
+        if command_name in ["left_delete", "commit_completion", "insert_best_completion", "replace_completion_with_next_completion", "toggle_comment", "insert_snippet"] or is_new_line_inserted:
             self._stop_completion = True
             return
 
-    def on_query_context(self, view, key, operator, operand, match_all): #pylint: disable=W0613
+    def on_query_context(self, view, key, operator, operand, match_all):  # pylint: disable=W0613
         if key == "tab_nine_choice_available":
             return False
         if key == "tab_nine_leader_key_available":
             return False
         if key == "tab_nine_reverse_leader_key_available":
             return False
```

Comment:
Could be truncated to `if key in ["tab_nine_choice_available",...]`, also it seems like it either returns `False` or `None`. Is this a wanted behaviour?

Reply:
yes, this is for backward compatibility with V1

Reply:
done